algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -964,7 +972,7 @@ class TopNCMSMonoid[K: Ordering](cms: CMS[K], heavyHittersN: Int = 100) extends
   /**
    * Creates a sketch out of a single item.
    */
-  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms, params)
+  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms + item, params)
```

Comment:
We do not have a direct test for this (may or may not be helpful, I am fine either way), but the new property-based tests for estimating inner product has been triggering the bug pretty reliably before ai fixed it.  The reason we didn't see this bug before was that the previous, non-fuzzy test for estimating inner products happened to use a test fixture (e.g. totalCount setting) so that the test passed kinda by pure luck.

Maybe we should actually add a direct test for this anyways. What do you think?